Week 2 Assignment 1/Fibonacci_W2A1_1.java

```diff
@@ -0,0 +1,18 @@
+public class Fibonacci_W2A1_1 {
+	static int temp1 = 0, temp2 = 1, temp3 = 0;
+	public static void main(String arg[]) {
+		int num = 9;
+		System.out.print(temp1 + " " + temp2 + " ");
+		recursion(num);
+	}
+	
+	static void recursion(int num) {
+		if(num > 0) {
+			temp3 = temp1 + temp2;
+			temp1 = temp2;
+			temp2 = temp3;
```

Comment:
Unnecessary variables used. Again, consider using the terminating condition to your benefit.